src/components/NFTViewCard.vue

```diff
@@ -46,6 +46,15 @@
         </div>
 
         <!--the rest-->
+        <!--Note: Ticket Type/Status set up with assumption they will be always fixed objects as 1st & 2nd indexes of attributes array -->
+        <p>
+          Ticket Type:
+          <span class="text-black">{{ n.metadataExternal.attributes[0].value ? n.metadataExternal.attributes[0].value : NOT_FOUND }}</span>
```

Comment:
we shouldnt assume its always attributes[0]. why cant we just look it up by its key (attributes.get("ticket_type")?

Reply:
i'll review this again/play with it, but i think the problem I ran into was that each dictionary/object in the attributes array has the same key:

Ex:


"attributes":
[
{
"trait_type":
"ticket_type",
"value":
"question"
},
{
"trait_type":
"status",
"value":
"open"
}
],

Reply:
i think i found a solution using filter() method:

`// using filter() method on an array
let x = {"attributes":
[
{
"trait_type":
"ticket_type",
"value":
"question"
},
{
"trait_type":
"status",
"value":
"open"
},
{
"trait_type":
"status",
"value":
"closed"
}
],
}


console.log(x.attributes.filter(x => x.trait_type == 'status'))

// get back: [{trait_type: "status", value: "open"}, {trait_type: "status", value: "closed"}]

`